src/dawn/CodeGen/CXXNaive-ico/CXXNaiveCodeGen.cpp

```diff
@@ -62,20 +62,20 @@ static std::string makeKLoop(const std::string dom, bool isBackward,
                     : makeLoopImpl(iir::Extent{}, "k", lower, upper, "<=", "++");
 }
 
-CXXNaiveCodeGen::CXXNaiveCodeGen(stencilInstantiationContext& ctx, DiagnosticsEngine& engine,
-                                 int maxHaloPoint)
+CXXNaiveIcoCodeGen::CXXNaiveIcoCodeGen(stencilInstantiationContext& ctx, DiagnosticsEngine& engine,
+                                       int maxHaloPoint)
     : CodeGen(ctx, engine, maxHaloPoint) {}
 
-CXXNaiveCodeGen::~CXXNaiveCodeGen() {}
+CXXNaiveIcoCodeGen::~CXXNaiveIcoCodeGen() {}
 
-std::string CXXNaiveCodeGen::generateStencilInstantiation(
+std::string CXXNaiveIcoCodeGen::generateStencilInstantiation(
     const std::shared_ptr<iir::StencilInstantiation> stencilInstantiation) {
   using namespace codegen;
 
   std::stringstream ssSW;
 
   Namespace dawnNamespace("dawn_generated", ssSW);
-  Namespace cxxnaiveNamespace("cxxnaive", ssSW);
+  Namespace cxxnaiveNamespace("cxxnaiveico", ssSW);
 
   const auto& globalsMap = stencilInstantiation->getIIR()->getGlobalVariableMap();
```

Comment:
should be commented out (along with `makeKLoop` and `makeIJLoop`), since it is not used and therefore gives compiler warning

Reply:
fixed warnings.